src/main/java/com/edmunds/rest/databricks/DTO/ClusterEventTypeDTO.java

```diff
@@ -22,22 +22,23 @@
  *
  */
 public enum ClusterEventTypeDTO implements Serializable {
-  CREATING("CREATING"),
-  STARTING("STARTING"),
-  RESTARTING("RESTARTING"),
-  TERMINATING("TERMINATING"),
-  EDITED("EDITED"),
-  RUNNING("RUNNING"),
-  RESIZING("RESIZING"),
-  NODES_LOST("NODES_LOST");
+    CREATING("CREATING"),
+    STARTING("STARTING"),
+    RESTARTING("RESTARTING"),
+    TERMINATING("TERMINATING"),
+    EDITED("EDITED"),
+    RUNNING("RUNNING"),
+    RESIZING("RESIZING"),
+    UPSIZE_COMPLETED("UPSIZE_COMPLETED"),
```

Comment:
Adding a case to an enum is potentially a breaking change, especially because this class is `Serializable`. You might want to rev-bump the pom and ensure that any code that deserializes this object allows for null/unknown/optional values.

Reply:
thanks for the explanation!

So these enum values are defined by the databricks rest api. Currently we were missing at least this one that I added (potentially more, which I suppose I might want to just go ahead and add now as part of this MR, I will follow up on that).

I am trying to think if there would actually be a breaking situation with how people use this, and I don't believe it would be the case. These values are only serialized/deserialized when making calls with the specific endpoints. I don't believe anybody would be storing these values to use on a different execution. @javamonkey79 what are your thoughts?

With that being said, I guess best practices for semantic versioning are to rev the middle version on a feature addition like this anyways, so I will go ahead and do that.

Reply:
While, (as you said) Sam, that it is unlikely for anyone to be storing, there's not harm in revving the version if not for consistency alone.